_posts/fr/newsletters/2022-09-14-newsletter.md

```diff
@@ -0,0 +1,171 @@
+---
+title: 'Bitcoin Optech Newsletter #217'
+permalink: /fr/newsletters/2022/09/14/
+name: 2022-09-14-newsletter-fr
+slug: 2022-09-14-newsletter-fr
+type: newsletters-fr
+layout: newsletters-fr
+lang: fr
+---
+La newsletter de cette semaine comprend notre section habituelle avec le
+résumé d'une réunion du Bitcoin Core PR Review Club une liste de nouvelles mises à jour des logiciels et des candidats
+à la publication, et des résumés de changements notables apportés aux principaux projets
```

Comment:
A propos les 2 expressions "candidats a la publication" : je suis pas certain que "publication" soit l'equivalent juste de "release", plutot :

- une release candidate
- une version candidate

Voir plus : 
- https://tr-ex.me/traduction/anglais-fran%C3%A7ais/release+candidate
- https://fr.glosbe.com/en/fr/release%20candidate

Reply:
une pré-version pourrait convenir ?

Reply:
> une pré-version pourrait convenir ?

Je ne sais pas :) entre "pré-version" et "release-candidate" ou "version candidate" je ne connais que les derniers -- les devs utilisaient pas mal de franglais mais ca fait quelques annees que je ne travaille plus en francais alors je ne suis plus tres au courant des pratiques.

Reply:
> 

Comment est-il possible de savoir ? Je suis parti sur "pré-version" qui me parait compréhensible et français

Reply:
Peut-etre demander a des collegues devs francais ou faire un sondage sur twitter.

Reply:
I don't think it's a blocker. The current version could be merged and you could make a later PR could update these terms if needed.

Reply:
je fais un sondage Twitter quand même pour les prochaines fois :-)
https://twitter.com/copinmalin/status/1574376615810932738?s=20&t=mnb7Cr-9VP2K250DqOajJQ

Reply:
> je fais un sondage Twitter quand même pour les prochaines fois :-) https://twitter.com/copinmalin/status/1574376615810932738?s=20&t=mnb7Cr-9VP2K250DqOajJQ

Cool, retweeted and voted.